src/components/seances/SeanceCard.vue

```diff
@@ -1,19 +1,22 @@
 <template>
   <div class="seance-card">
     <div class="seance-card__poster">
-      <img :src="matchedMovie(movieId).poster_url" alt="poster" />
+      <router-link :to="`movie/${movieId}`">
```

Comment:
If you have named routes, it's worth it to consider using names instead of paths, i.e.
```suggestion
      <router-link :to="{ name: 'MovieDetailsPage', params: { movieId } }">
```
This way we do not need to repeat the paths, and if they change - we do not need to change code in other places (for example here 😄).

Reply:
thank you!!! :)